docs/guides/30_dockerfile.adoc

```diff
@@ -223,4 +223,4 @@ RUN ["jlink", "--compress=2", "--strip-debug", "--no-header-files", \
      "--add-modules", \
      "java.base,java.logging,java.sql,java.desktop,java.management,java.naming,jdk.unsupported", \
      "--output", "/var/tmp/myjre"]
-----
+----
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"